code/app/build.gradle.kts

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import com.adobe.marketing.mobile.gradle.BuildConstants
+
+plugins {
+    id("com.android.application")
+    id("com.diffplug.spotless")
+}
+
+val mavenCoreVersion: String by project
+
+configure<com.diffplug.gradle.spotless.SpotlessExtension> {
+    java {
+        toggleOffOn("format:off", "format:on")
+        target("src/*/java/**/*.java")
+        removeUnusedImports()
+        prettier(mapOf("prettier" to "2.7.1", "prettier-plugin-java" to "1.6.2"))
+        .config(mapOf("parser" to "java", "tabWidth" to 4, "useTabs" to true, "printWidth" to 120))
+        endWithNewline()
+        licenseHeader(BuildConstants.ADOBE_LICENSE_HEADER)
+    }
+}
+
+android {
+    namespace = "com.adobe.marketing.tester"
```

Comment:
we need to refactor the app name space to unique namespace
com.adobe.marketing.mobile.<extension>.testapp

Reply:
The namespace matches the directory structure, so I'll need to move and update all the test app files. I'll do this in another PR.